frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -174,6 +179,21 @@ private ConfigManager(Arguments args) {
         }
     }
 
+    private void resolveEnvVarVals(Properties prop) {
+        Set<String> keys = prop.stringPropertyNames();
+        for (String key : keys) {
+            String val = prop.getProperty(key);
+            Matcher matcher = pattern.matcher(val);
+            if (matcher.find()) {
```

Comment:
macher.matches(), and you can remove ^ and $ in your regex.

Reply:
Sure.